test/TestProjects/ExactMatchFlattenInheritance/Generated/LongRunningOperation/ExactMatchFlattenInheritanceArmOperation.cs

```diff
@@ -29,11 +29,13 @@ protected ExactMatchFlattenInheritanceArmOperation()
         internal ExactMatchFlattenInheritanceArmOperation(Response response)
         {
             _operation = new OperationOrResponseInternals(response);
+            _operation.PollingStrategy = new ExponentialPollingStrategy();
```

Comment:
Another options is that we add constructors of `OperationOrResponseInternals` for mgmt plane. And invoke the specific constructor here.